src/main/java/spark/utils/GzipUtils.java

```diff
@@ -68,8 +68,10 @@ public static OutputStream checkAndWrap(HttpServletRequest httpRequest,
 
         if (acceptsGzip) {
             if (!requireWantsHeader || wantGzip) {
+            	if(spark.Spark.doGzip==true){
```

Comment:
if you are accessing something in a some other class/object use a getter. IMO this is very confusing.